packages/vue-query/src/vueQueryPlugin.ts

```diff
@@ -59,6 +59,14 @@ export const VueQueryPlugin = {
 
     client.mount()
 
+    if (process.env.NODE_ENV !== 'production' && options.contextSharing) {
+      client
+        .getLogger()
+        .error(
+          `The contextSharing option has been deprecated and will be removed in the next major version`,
+        )
+    }
+
```

Comment:
@DamianOsipiuk please let me know if this is the right place to do this for vue-query

Reply:
hmm for some reason, I'm getting an error that says `client.getLogger()` doesn't work. Do you know what I'm doing wrong?

```
packages/vue-query test:jest:   ● VueQueryPlugin › when context sharing is enabled › should use existing context
packages/vue-query test:jest:     TypeError: client.getLogger is not a function
packages/vue-query test:jest:       62 |     if (process.env.NODE_ENV !== 'production' && options.contextSharing) {
packages/vue-query test:jest:       63 |       client
packages/vue-query test:jest:     > 64 |         .getLogger()
packages/vue-query test:jest:          |          ^
packages/vue-query test:jest:       65 |         .error(
packages/vue-query test:jest:       66 |           `The contextSharing option has been deprecated and will be removed in the next major version`,
packages/vue-query test:jest:       67 |         )
```

Reply:
It's due to this mock, which has only one method and is not a real queryClient
https://github.com/TanStack/query/blob/1fe93566c43dd10d4f4adf307bfe337b6fc01a09/packages/vue-query/src/__tests__/vueQueryPlugin.test.ts#L248

Reply:
thx - fixed it here: 5f7ada6a